### PR TITLE
fix php_gd missing

### PR DESCRIPTION
**Pull request by remyslab**
Hi, there is a fix for the "comment error" (GD extension missing).

### CODE REVIEW

**Comment by tomas-samot** [comment]
Sorry I didn't see your PR, I will review it now. @remyslab

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,17 @@ RUN chmod a+x /cert.sh
...
 
-RUN apt-get update
+RUN apt-get update && apt-get install -y \
+        libfreetype6-dev \
+        libjpeg62-turbo-dev \
+        libpng12-dev \
+        wget \
+        zip \
+        unzip; \
+    # php-gd config
```
Comment (tomas-samot) [inline Dockerfile]: Comment needs to be removed.

File: Dockerfile [hunk 2]
```diff
@@ -10,7 +10,17 @@ RUN chmod a+x /cert.sh
...
 RUN sed -i "/#ServerName www.example.com/c\ServerName $DOMAIN" /etc/apache2/sites-available/000-default.conf
 
-RUN apt-get update
+RUN apt-get update && apt-get install -y \
+        libfreetype6-dev \
+        libjpeg62-turbo-dev \
+        libpng12-dev \
+        wget \
+        zip \
+        unzip; \
```
Comment (tomas-samot) [inline Dockerfile]: This is repeated below.

File: Dockerfile [hunk 3]
```diff
@@ -10,7 +10,17 @@ RUN chmod a+x /cert.sh
 
 RUN sed -i "/#ServerName www.example.com/c\ServerName $DOMAIN" /etc/apache2/sites-available/000-default.conf
 
-RUN apt-get update
+RUN apt-get update && apt-get install -y \
+        libfreetype6-dev \
+        libjpeg62-turbo-dev \
+        libpng12-dev \
+        wget \
```
Comment (tomas-samot) [inline Dockerfile]: This is repeated below.

File: Dockerfile [hunk 4]
```diff
@@ -10,7 +10,17 @@ RUN chmod a+x /cert.sh
 
 RUN sed -i "/#ServerName www.example.com/c\ServerName $DOMAIN" /etc/apache2/sites-available/000-default.conf
 
-RUN apt-get update
+RUN apt-get update && apt-get install -y \
```
Comment (tomas-samot) [inline Dockerfile]: Please separate the update and install.

**Comment by tomas-samot** [comment]
Why are the following packages needed?
```
libfreetype6-dev
libjpeg62-turbo-dev
libpng12-dev
zip
```

**Comment by remyslab** [comment]
Hi @tomas-samot, thanks for your feedbacks. Sorry if it's a bit messy, I'll clean it and make a PR again. 
Without this :
```
libfreetype6-dev
libjpeg62-turbo-dev
libpng12-dev
```
I have some error for avatars display like `configure: error: jpeglib.h not found.`
I simply adapted [the original code proposed by the initial repo](https://github.com/PrivateBin/PrivateBin/commit/f45367fac2b5b9405c23f6867c551962122bc970).
`zip` doesn't seem be required, I will delete this line.

**Review by tomas-samot — APPROVED** [review]

**Comment by tomas-samot** [comment]
@remyslab Thank you for your contribution.

**Pull request merged**